content/src/display-content/business-formation/nexus/form-business-entity.md

```diff
@@ -8,14 +8,12 @@ callToActionLink: https://www.njportal.com/DOR/BusinessFormation/CompanyInformat
 callToActionText: Authorize My Business
 required: true
 agencyId: nj-revenue-enterprise-services
+summaryDescriptionMd: >-
+  You must register your business name and authorize your `business structure|business-structure-learn-more`
+  with the State.
+bodyHeading: Application Requirements
 ---
```

Comment:
[dust]: Would this create different page content? With the previous ones the body header was above the summaryDescription but with this one it was below in the diff.